v0.1/core/annotation.schema.json

```diff
@@ -4,6 +4,13 @@
   "description": "Definition of an annotation",
   "type": "object",
   "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "criteriaType",
+    "laterality",
+    "visualizedIn"
+  ],
```

Comment:
Issue #27 has `criteriaType`, `definingTerm`, `laterality`and `visualizedIn`defined as required. `definingTerm` is missing

Reply:
yes. I just forgot it.